pkgs/development/libraries/libprotobuf-mutator/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, clangStdenv
+, cmake
+, fetchFromGitHub
+, git
+, libtool
+, lzma
+, ninja
+, pkg-config
+, protobuf
+, zlib
+}:
+
+let
+  # See `src/cmake/external/googletest.cmake`.
+  googletest_src = fetchFromGitHub {
```

Comment:
Can we override nixpkgs gtest?

Reply:
I am not too sure what you mean by that...

Reply:
Can we re-use https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/gtest/default.nix ?